### ISSUE TITLE
Problem with Mocha

### ISSUE THREAD

**Issue by KittyGiraudel** [HugoGiraudel/node-legofy#22]
## Setup
- Branch: `master`
- Node version: `v0.12.7`
- npm version: `2.11.3`
- Fresh modules install
## Stack trace

```
$ npm test

> node-legofy@0.1.2 test /Users/hgiraudel/Sites/node-legofy
> mocha test --compilers js:babel-core/register

module.js:338
    throw err;
          ^
Error: Cannot find module 'array-from'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (index.js:63:31)
    at Module._compile (module.js:460:26)
    at loader (/Users/hgiraudel/Sites/node-legofy/node_modules/babel-core/lib/api/register/node.js:136:5)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/hgiraudel/Sites/node-legofy/node_modules/babel-core/lib/api/register/node.js:146:7)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/hgiraudel/Sites/node-legofy/test/index.js:3:12)
    at Module._compile (module.js:460:26)
    at loader (/Users/hgiraudel/Sites/node-legofy/node_modules/babel-core/lib/api/register/node.js:136:5)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/hgiraudel/Sites/node-legofy/node_modules/babel-core/lib/api/register/node.js:146:7)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at /Users/hgiraudel/Sites/node-legofy/node_modules/mocha/lib/mocha.js:216:27
    at Array.forEach (native)
    at Mocha.loadFiles (/Users/hgiraudel/Sites/node-legofy/node_modules/mocha/lib/mocha.js:213:14)
    at Mocha.run (/Users/hgiraudel/Sites/node-legofy/node_modules/mocha/lib/mocha.js:453:10)
    at Object.<anonymous> (/Users/hgiraudel/Sites/node-legofy/node_modules/mocha/bin/_mocha:393:18)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
    at startup (node.js:129:16)
    at node.js:814:3
npm ERR! Test failed.  See above for more details.
```

**Comment by KittyGiraudel** [comment]
Should be fixed with https://github.com/valeriangalliat/babel-plugin-transform-array-from/pull/1.

**Comment by valeriangalliat** [comment]
Fixed with #23
